### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#487] [Esami & Esoneri M-Z]
### Traccia

<img width="589" alt="image" src="https://github.com/user-attachments/assets/1d176ffe-4388-41d5-bb2d-2de2a72c1d47">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/e5f61748-f4f6-40f9-a0df-7c707c65ede3)

## Tableau

![image](https://github.com/user-attachments/assets/26fa5e34-1f18-470f-b1e0-69e1533c126e)
